src/Menu/ExpandMacros.cpp

```diff
@@ -80,26 +80,26 @@ ExpandTaskMacros(const TCHAR *name,
     if (StringIsEqual(name, _T("WaypointNext")) ||
         StringIsEqual(name, _T("WaypointNextArm"))) {
       invalid = true;
-      return _("Next Turnpoint");
+      return _("Turnpoint Next");
```

Comment:
This is awkward. "Turnpoint Next" - what does that mean? I don't like swapping those words, because the label doesn't make sense that way, it's confusing and thus wastes the user's attention.